# Journal Club Publication List
---
> ### Improving data workflow systems with cloud services and use of open data for bioinformatics research.
> ###### Briefings in bioinformatics, Date: 4/2017

> #### Karim Md Rezaul, Michel Audrey, Zappa Achille, Baranov Pavel, Sahay Ratnesh, Rebholz-Schuhmann Dietrich
> ##### Data workflow systems (DWFSs) enable bioinformatics researchers to combine components for data access and data analytics, and to share the final data analytics approach with their collaborators. Increasingly, such systems have to cope with large-scale data, such as full genomes (about 200 GB each), public fact repositories (about 100 TB of data) and 3D imaging data at even larger scales. As moving the data becomes cumbersome, the DWFS needs to embed its processes into a cloud infrastructure, where the data are already hosted. As the standardized public data play an increasingly important role, the DWFS needs to comply with Semantic Web technologies. This advancement to DWFS would reduce overhead costs and accelerate the progress in bioinformatics research based on large-scale data and public resources, as researchers would require less specialized IT knowledge for the implementation. Furthermore, the high data growth rates in bioinformatics research drive the demand for parallel and distributed computing, which then imposes a need for scalability and high-throughput capabilities onto the DWFS. As a result, requirements for data sharing and access to public knowledge bases suggest that compliance of the DWFS with Semantic Web standards is necessary. In this article, we will analyze the existing DWFS with regard to their capabilities toward public open data use as well as large-scale computational and human interface requirements. We untangle the parameters for selecting a preferable solution for bioinformatics research with particular consideration to using cloud services and Semantic Web technologies. Our analysis leads to research guidelines and recommendations toward the development of future DWFS for the bioinformatics research community.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/22644661)

---
> ### BioStructMap: a Python tool for integration of protein structure and sequence-based features.
> ###### Bioinformatics (Oxford, England), Date: 6/2018

> #### Guy Andrew J, Irani Vashti, Richards Jack S, Ramsland Paul A
> ##### Supplementary data are available at Bioinformatics online.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/20478830)

---
> ### tRNA epitranscriptomics and biased codon are linked to proteome expression in 
> ###### Molecular systems biology, Date: 10/2018

> #### Ng Chee Sheng, Sinha Ameya, Aniweh Yaw, Nah Qianhui, Babu Indrakanti Ramesh, Gu Chen, Chionh Yok Hian, Dedon Peter C, Preiser Peter R
> ##### Among components of the translational machinery, ribonucleoside modifications on tRNAs are emerging as critical regulators of cell physiology and stress response. Here, we demonstrate highly coordinated behavior of the repertoire of tRNA modifications of 

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/21187895)

---
> ### Multi-omic and multi-view clustering algorithms: review and cancer benchmark.
> ###### Nucleic acids research, Date: 10/2018

> #### Rappoport Nimrod, Shamir Ron
> ##### Recent high throughput experimental methods have been used to collect large biomedical omics datasets. Clustering of single omic datasets has proven invaluable for biological and medical research. The decreasing cost and development of additional high throughput methods now enable measurement of multi-omic data. Clustering multi-omic data has the potential to reveal further systems-level insights, but raises computational and biological challenges. Here, we review algorithms for multi-omics clustering, and discuss key issues in applying these algorithms. Our review covers methods developed specifically for omic data as well as generic multi-view methods developed in the machine learning community for joint clustering of multiple data types. In addition, using cancer data from TCGA, we perform an extensive benchmark spanning ten different cancer types, providing the first systematic comparison of leading multi-omics and multi-view clustering algorithms. The results highlight key issues regarding the use of single- versus multi-omics, the choice of clustering strategy, the power of generic multi-view methods and the use of approximated p-values for gauging solution quality. Due to the growing use of multi-omics data, we expect these issues to be important for future progress in the field.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/26353207)

---
> ### QstR-dependent regulation of natural competence and type VI secretion in Vibrio cholerae.
> ###### Nucleic acids research, Date: 8/2018

> #### Jaskólska Milena, Stutzmann Sandrine, Stoudmann Candice, Blokesch Melanie
> ##### During growth on chitinous surfaces in its natural aquatic environment Vibrio cholerae develops natural competence for transformation and kills neighboring non-immune bacteria using a type VI secretion system (T6SS). Activation of these two phenotypes requires the chitin-induced regulator TfoX, but also integrates signals from quorum sensing via the intermediate regulator QstR, which belongs to the LuxR-type family of regulators. Here, we define the QstR regulon using RNA sequencing. Moreover, by mapping QstR binding sites using chromatin immunoprecipitation coupled with deep sequencing we demonstrate that QstR is a transcription factor that binds upstream of the up- and down-regulated genes. Like other LuxR-type family transcriptional regulators we show that QstR function is dependent on dimerization. However, in contrast to the well-studied LuxR-type biofilm regulator VpsT of V. cholerae, which requires the second messenger c-di-GMP, we show that QstR dimerization and function is c-di-GMP independent. Surprisingly, although ComEA, which is a periplasmic DNA-binding protein essential for transformation, is produced in a QstR-dependent manner, QstR-binding was not detected upstream of comEA suggesting the existence of a further regulatory pathway. Overall, these results provide detailed insights into the function of a key regulator of natural competence and type VI secretion in V. cholerae.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/22222000)

---
> ### Endogenous single-strand DNA breaks at RNA polymerase II promoters in Saccharomyces cerevisiae.
> ###### Nucleic acids research, Date: 11/2018

> #### Hegedüs Éva, Kókai Endre, Nánási Péter, Imre László, Halász László, Jossé Rozenn, Antunovics Zsuzsa, Webb Martin R, El Hage Aziz, Pommier Yves, Székvölgyi Lóránt, Dombrádi Viktor, Szabó Gábor
> ##### Molecular combing and gel electrophoretic studies revealed endogenous nicks with free 3'OH ends at ∼100 kb intervals in the genomic DNA (gDNA) of unperturbed and G1-synchronized Saccharomyces cerevisiae cells. Analysis of the distribution of endogenous nicks by Nick ChIP-chip indicated that these breaks accumulated at active RNA polymerase II (RNAP II) promoters, reminiscent of the promoter-proximal transient DNA breaks of higher eukaryotes. Similar periodicity of endogenous nicks was found within the ribosomal rDNA cluster, involving every ∼10th of the tandemly repeated 9.1 kb units of identical sequence. Nicks were mapped by Southern blotting to a few narrow regions within the affected units. Three of them were overlapping the RNAP II promoters, while the ARS-containing IGS2 region was spared of nicks. By using a highly sensitive reverse-Southwestern blot method to map free DNA ends with 3'OH, nicks were shown to be distinct from other known rDNA breaks and linked to the regulation of rDNA silencing. Nicks in rDNA and the rest of the genome were typically found at the ends of combed DNA molecules, occasionally together with R-loops, comprising a major pool of vulnerable sites that are connected with transcriptional regulation.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/24743386)

---
