# Journal Club Publication List
---
> ### 1- Combining probabilistic alignments with read pair information improves accuracy of split-alignments.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Shrestha Anish M S
> ##### Split-alignments provide base-pair-resolution evidence of genomic rearrangements. In practice, they are found by first computing high-scoring local alignments, parts of which are then combined into a split-alignment. This approach is challenging when aligning a short read to a large and repetitive reference, as it tends to produce many spurious local alignments leading to ambiguities in identifying the correct split-alignment. This problem is further exacerbated by the fact that rearrangements tend to occur in repeat-rich regions.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29790902), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY398)

---
> ### 2- Development and evaluation of a deep learning model for protein-ligand binding affinity prediction.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Stepniewska-Dziubinska Marta M
> ##### Structure based ligand discovery is one of the most successful approaches for augmenting the drug discovery process. Currently, there is a notable shift towards machine learning (ML) methodologies to aid such procedures. Deep learning has recently gained considerable attention as it allows the model to 'learn' to extract features that are relevant for the task at hand.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29757353), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY374)

---
> ### 3- PyCoTools: a Python toolbox for COPASI.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Welsh Ciaran M
> ##### COPASI is an open source software package for constructing, simulating and analyzing dynamic models of biochemical networks. COPASI is primarily intended to be used with a graphical user interface but often it is desirable to be able to access COPASI features programmatically, with a high level interface.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29790940), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY409)

---
> ### 4- ConSpeciFix: classifying prokaryotic species based on gene flow.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Bobay Louis-Marie
> ##### Classification of prokaryotic species is usually based on sequence similarity thresholds, which are easy to apply but lack a biologically-relevant foundation. Here, we present ConSpeciFix, a program that classifies prokaryotes into species using criteria set forth by the Biological Species Concept, thereby unifying species definition in all domains of life.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29771275), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY400)

---
> ### 5- SoS Notebook: an interactive multi-language data analysis environment.
> ###### Bioinformatics (Oxford, England), Date: 10/2018

> #### Peng Bo
> ##### Complex bioinformatic data analysis workflows involving multiple scripts in different languages can be difficult to consolidate, share and reproduce. An environment that streamlines the entire processes of data collection, analysis, visualization and reporting of such multi-language analyses is currently lacking.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29790910), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY405)

---
> ### 6- SciApps: a cloud-based platform for reproducible bioinformatics workflows.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Wang Liya
> ##### The rapid accumulation of both sequence and phenotype data generated by high-throughput methods has increased the need to store and analyze data on distributed storage and computing systems. Efficient data management across these heterogeneous systems requires a workflow management system to simplify the task of analysis through automation and make large-scale bioinformatics analyses accessible and reproducible.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29897418), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY439)

---
> ### 7- Motif scraper: a cross-platform, open-source tool for identifying degenerate nucleotide motif matches in FASTA files.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Roberson Elisha D O
> ##### Many genomic features are defined not by exact sequence matches, but by degenerate nucleotide motifs that represent multiple compatible matches. While there are databases cataloging genomic features, such as the location of transcription factor motifs, for commonly used model species, identifying the locations of novel motifs, known motifs in non-model genomes, or known motifs in personal whole-genomes is difficult. I designed motif scraper to overcome this limitation, allowing for efficient, multiprocessor motif searches in any FASTA file.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29850891), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY437)

---
> ### 8- Improving data workflow systems with cloud services and use of open data for bioinformatics research.
> ###### Briefings in bioinformatics, Date: 11/2018

> #### Karim Md Rezaul
> ##### Data workflow systems (DWFSs) enable bioinformatics researchers to combine components for data access and data analytics, and to share the final data analytics approach with their collaborators. Increasingly, such systems have to cope with large-scale data, such as full genomes (about 200 GB each), public fact repositories (about 100 TB of data) and 3D imaging data at even larger scales. As moving the data becomes cumbersome, the DWFS needs to embed its processes into a cloud infrastructure, where the data are already hosted. As the standardized public data play an increasingly important role, the DWFS needs to comply with Semantic Web technologies. This advancement to DWFS would reduce overhead costs and accelerate the progress in bioinformatics research based on large-scale data and public resources, as researchers would require less specialized IT knowledge for the implementation. Furthermore, the high data growth rates in bioinformatics research drive the demand for parallel and distributed computing, which then imposes a need for scalability and high-throughput capabilities onto the DWFS. As a result, requirements for data sharing and access to public knowledge bases suggest that compliance of the DWFS with Semantic Web standards is necessary. In this article, we will analyze the existing DWFS with regard to their capabilities toward public open data use as well as large-scale computational and human interface requirements. We untangle the parameters for selecting a preferable solution for bioinformatics research with particular consideration to using cloud services and Semantic Web technologies. Our analysis leads to research guidelines and recommendations toward the development of future DWFS for the bioinformatics research community.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/28419324), [DOI](http://doi.org/10.1093/BIB/BBX039)

---
> ### 9- BioStructMap: a Python tool for integration of protein structure and sequence-based features.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Guy Andrew J
> ##### A sliding window analysis over a protein or genomic sequence is commonly performed, and we present a Python tool, BioStructMap, that extends this concept to three-dimensional (3D) space, allowing the application of a 3D sliding window analysis over a protein structure. BioStructMap is easily extensible, allowing the user to apply custom functions to spatially aggregated data. BioStructMap also allows mapping of underlying genomic sequences to protein structures, allowing the user to perform genetic-based analysis over spatially linked codons-this has applications when selection pressures arise at the level of protein structure.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29931276), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY474)

---
> ### 10- tRNA epitranscriptomics and biased codon are linked to proteome expression in 
> ###### Molecular systems biology, Date: 11/2018

> #### Ng Chee Sheng
> ##### Among components of the translational machinery, ribonucleoside modifications on tRNAs are emerging as critical regulators of cell physiology and stress response. Here, we demonstrate highly coordinated behavior of the repertoire of tRNA modifications of 

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30287681), [DOI](http://doi.org/10.15252/MSB.20178009)

---
> ### 11- Multi-omic and multi-view clustering algorithms: review and cancer benchmark.
> ###### Nucleic acids research, Date: 11/2018

> #### Rappoport Nimrod
> ##### Recent high throughput experimental methods have been used to collect large biomedical omics datasets. Clustering of single omic datasets has proven invaluable for biological and medical research. The decreasing cost and development of additional high throughput methods now enable measurement of multi-omic data. Clustering multi-omic data has the potential to reveal further systems-level insights, but raises computational and biological challenges. Here, we review algorithms for multi-omics clustering, and discuss key issues in applying these algorithms. Our review covers methods developed specifically for omic data as well as generic multi-view methods developed in the machine learning community for joint clustering of multiple data types. In addition, using cancer data from TCGA, we perform an extensive benchmark spanning ten different cancer types, providing the first systematic comparison of leading multi-omics and multi-view clustering algorithms. The results highlight key issues regarding the use of single- versus multi-omics, the choice of clustering strategy, the power of generic multi-view methods and the use of approximated p-values for gauging solution quality. Due to the growing use of multi-omics data, we expect these issues to be important for future progress in the field.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30295871), [DOI](http://doi.org/10.1093/NAR/GKY889)

---
> ### 12- QstR-dependent regulation of natural competence and type VI secretion in Vibrio cholerae.
> ###### Nucleic acids research, Date: 11/2018

> #### Jaskólska Milena
> ##### During growth on chitinous surfaces in its natural aquatic environment Vibrio cholerae develops natural competence for transformation and kills neighboring non-immune bacteria using a type VI secretion system (T6SS). Activation of these two phenotypes requires the chitin-induced regulator TfoX, but also integrates signals from quorum sensing via the intermediate regulator QstR, which belongs to the LuxR-type family of regulators. Here, we define the QstR regulon using RNA sequencing. Moreover, by mapping QstR binding sites using chromatin immunoprecipitation coupled with deep sequencing we demonstrate that QstR is a transcription factor that binds upstream of the up- and down-regulated genes. Like other LuxR-type family transcriptional regulators we show that QstR function is dependent on dimerization. However, in contrast to the well-studied LuxR-type biofilm regulator VpsT of V. cholerae, which requires the second messenger c-di-GMP, we show that QstR dimerization and function is c-di-GMP independent. Surprisingly, although ComEA, which is a periplasmic DNA-binding protein essential for transformation, is produced in a QstR-dependent manner, QstR-binding was not detected upstream of comEA suggesting the existence of a further regulatory pathway. Overall, these results provide detailed insights into the function of a key regulator of natural competence and type VI secretion in V. cholerae.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30102403), [DOI](http://doi.org/10.1093/NAR/GKY717)

---
> ### 13- Endogenous single-strand DNA breaks at RNA polymerase II promoters in Saccharomyces cerevisiae.
> ###### Nucleic acids research, Date: 11/2018

> #### Hegedüs Éva
> ##### Molecular combing and gel electrophoretic studies revealed endogenous nicks with free 3'OH ends at ∼100 kb intervals in the genomic DNA (gDNA) of unperturbed and G1-synchronized Saccharomyces cerevisiae cells. Analysis of the distribution of endogenous nicks by Nick ChIP-chip indicated that these breaks accumulated at active RNA polymerase II (RNAP II) promoters, reminiscent of the promoter-proximal transient DNA breaks of higher eukaryotes. Similar periodicity of endogenous nicks was found within the ribosomal rDNA cluster, involving every ∼10th of the tandemly repeated 9.1 kb units of identical sequence. Nicks were mapped by Southern blotting to a few narrow regions within the affected units. Three of them were overlapping the RNAP II promoters, while the ARS-containing IGS2 region was spared of nicks. By using a highly sensitive reverse-Southwestern blot method to map free DNA ends with 3'OH, nicks were shown to be distinct from other known rDNA breaks and linked to the regulation of rDNA silencing. Nicks in rDNA and the rest of the genome were typically found at the ends of combed DNA molecules, occasionally together with R-loops, comprising a major pool of vulnerable sites that are connected with transcriptional regulation.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30445637), [DOI](http://doi.org/10.1093/NAR/GKY743)

---
> ### 14- Systematic discovery of uncharacterized transcription factors in Escherichia coli K-12 MG1655.
> ###### Nucleic acids research, Date: 11/2018

> #### Gao Ye
> ##### Transcriptional regulation enables cells to respond to environmental changes. Of the estimated 304 candidate transcription factors (TFs) in Escherichia coli K-12 MG1655, 185 have been experimentally identified, but ChIP methods have been used to fully characterize only a few dozen. Identifying these remaining TFs is key to improving our knowledge of the E. coli transcriptional regulatory network (TRN). Here, we developed an integrated workflow for the computational prediction and comprehensive experimental validation of TFs using a suite of genome-wide experiments. We applied this workflow to (i) identify 16 candidate TFs from over a hundred uncharacterized genes; (ii) capture a total of 255 DNA binding peaks for ten candidate TFs resulting in six high-confidence binding motifs; (iii) reconstruct the regulons of these ten TFs by determining gene expression changes upon deletion of each TF and (iv) identify the regulatory roles of three TFs (YiaJ, YdcI, and YeiE) as regulators of l-ascorbate utilization, proton transfer and acetate metabolism, and iron homeostasis under iron-limited conditions, respectively. Together, these results demonstrate how this workflow can be used to discover, characterize, and elucidate regulatory functions of uncharacterized TFs in parallel.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30137486), [DOI](http://doi.org/10.1093/NAR/GKY752)

---
> ### 15- Post-transcriptional gene regulation by an Hfq-independent small RNA in Caulobacter crescentus.
> ###### Nucleic acids research, Date: 11/2018

> #### Fröhlich Kathrin S
> ##### Bacterial small RNAs (sRNAs) are a heterogeneous group of post-transcriptional regulators that often act at the heart of large networks. Hundreds of sRNAs have been discovered by genome-wide screens and most of these sRNAs exert their functions by base-pairing with target mRNAs. However, studies addressing the molecular roles of sRNAs have been largely confined to gamma-proteobacteria, such as Escherichia coli. Here we identify and characterize a novel sRNA, ChvR, from the alpha-proteobacterium Caulobacter crescentus. Transcription of chvR is controlled by the conserved two-component system ChvI-ChvG and it is expressed in response to DNA damage, low pH, and growth in minimal medium. Transient over-expression of ChvR in combination with genome-wide transcriptome profiling identified the mRNA of the TonB-dependent receptor ChvT as the sole target of ChvR. Genetic and biochemical analyses showed that ChvR represses ChvT at the post-transcriptional level through direct base-pairing. Fine-mapping of the ChvR-chvT interaction revealed the requirement of two distinct base-pairing sites for full target regulation. Finally, we show that ChvR-controlled repression of chvT is independent of the ubiquitous RNA-chaperone Hfq, and therefore distinct from previously reported mechanisms employed by prototypical bacterial sRNAs. These findings have implications for the mechanism and evolution of sRNA function across bacterial species.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30165530), [DOI](http://doi.org/10.1093/NAR/GKY765)

---
> ### 16- Small RNA sequences derived from pre-microRNAs in the supraspliceosome.
> ###### Nucleic acids research, Date: 11/2018

> #### Mahlab-Aviv Shelly
> ##### MicroRNAs (miRNAs) are short non-coding RNAs that negatively regulate the expression and translation of genes in healthy and diseased tissues. Herein, we characterize short RNAs from human HeLa cells found in the supraspliceosome, a nuclear dynamic machine in which pre-mRNA processing occurs. We sequenced small RNAs (<200 nt) extracted from the supraspliceosome, and identified sequences that are derived from 200 miRNAs genes. About three quarters of them are mature miRNAs, whereas the rest account for various defined regions of the pre-miRNA, and its hairpin-loop precursor. Out of these aligned sequences, 53 were undetected in cellular extract, and the abundance of additional 48 strongly differed from that in cellular extract. Notably, we describe seven abundant miRNA-derived sequences that overlap non-coding exons of their host gene. The rich collection of sequences identical to pre-miRNAs at the supraspliceosome suggests overlooked nuclear functions. Specifically, the abundant hsa-mir-99b may affect splicing of LINC01129 primary transcript through base-pairing with its exon-intron junction. Using suppression and overexpression experiments, we show that hsa-mir-7704 negatively regulates the level of the lncRNA HAGLR. We claim that in cases of extended base-pairing complementarity, such supraspliceosomal pre-miRNA sequences might have a role in transcription attenuation, maturation and processing.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30203035), [DOI](http://doi.org/10.1093/NAR/GKY791)

---
> ### 17- Role of ribosome assembly in Escherichia coli ribosomal RNA degradation.
> ###### Nucleic acids research, Date: 11/2018

> #### Jain Chaitanya
> ##### DEAD-Box proteins (DBPs) constitute a prominent class of RNA remodeling factors that play a role in virtually all aspects of RNA metabolism. To better define their cellular functions, deletions in the genes encoding each of the Escherichia coli DBPs were combined with mutations in genes encoding different Ribonucleases (RNases). Significantly, double-deletion strains lacking Ribonuclease R (RNase R) and either the DeaD or SrmB DBP were found to display growth defects and an enhanced accumulation of ribosomal RNA (rRNA) fragments. As RNase R is known to play a key role in removing rRNA degradation products, these observations initially suggested that these two DBPs could be directly involved in the same process. However, additional investigations indicated that DeaD and SrmB-dependent rRNA breakdown is caused by delays in ribosome assembly that increase the exposure of nascent RNAs to endonucleolytic cleavage. Consistent with this notion, mutations in factors known to be important for ribosome assembly also resulted in enhanced rRNA breakdown. Additionally, significant levels of rRNA breakdown products could be visualized in growing cells even in the absence of assembly defects. These findings reveal a hitherto unappreciated mechanism of rRNA degradation under conditions of both normal and abnormal ribosome assembly.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30219894), [DOI](http://doi.org/10.1093/NAR/GKY808)

---
> ### 18- Disentangling transcription factor binding site complexity.
> ###### Nucleic acids research, Date: 11/2018

> #### Eggeling Ralf
> ##### The binding motifs of many transcription factors (TFs) comprise a higher degree of complexity than a single position weight matrix model permits. Additional complexity is typically taken into account either as intra-motif dependencies via more sophisticated probabilistic models or as heterogeneities via multiple weight matrices. However, both orthogonal approaches have limitations when learning from in vivo data where binding sites of other factors in close proximity can interfere with motif discovery for the protein of interest. In this work, we demonstrate how intra-motif complexity can, purely by analyzing the statistical properties of a given set of TF-binding sites, be distinguished from complexity arising from an intermix with motifs of co-binding TFs or other artifacts. In addition, we study the related question whether intra-motif complexity is represented more effectively by dependencies, heterogeneities or variants in between. Benchmarks demonstrate the effectiveness of both methods for their respective tasks and applications on motif discovery output from recent tools detect and correct many undesirable artifacts. These results further suggest that the prevalence of intra-motif dependencies may have been overestimated in previous studies on in vivo data and should thus be reassessed.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30085218), [DOI](http://doi.org/10.1093/NAR/GKY683)

---
> ### 19- TranSurVeyor: an improved database-free algorithm for finding non-reference transpositions in high-throughput sequencing data.
> ###### Nucleic acids research, Date: 11/2018

> #### Rajaby Ramesh
> ##### Transpositions transfer DNA segments between different loci within a genome; in particular, when a transposition is found in a sample but not in a reference genome, it is called a non-reference transposition. They are important structural variations that have clinical impact. Transpositions can be called by analyzing second generation high-throughput sequencing datasets. Current methods follow either a database-based or a database-free approach. Database-based methods require a database of transposable elements. Some of them have good specificity; however this approach cannot detect novel transpositions, and it requires a good database of transposable elements, which is not yet available for many species. Database-free methods perform de novo calling of transpositions, but their accuracy is low. We observe that this is due to the misalignment of the reads; since reads are short and the human genome has many repeats, false alignments create false positive predictions while missing alignments reduce the true positive rate. This paper proposes new techniques to improve database-free non-reference transposition calling: first, we propose a realignment strategy called one-end remapping that corrects the alignments of reads in interspersed repeats; second, we propose a SNV-aware filter that removes some incorrectly aligned reads. By combining these two techniques and other techniques like clustering and positive-to-negative ratio filter, our proposed transposition caller TranSurVeyor shows at least 3.1-fold improvement in terms of F1-score over existing database-free methods. More importantly, even though TranSurVeyor does not use databases of prior information, its performance is at least as good as existing database-based methods such as MELT, Mobster and Retroseq. We also illustrate that TranSurVeyor can discover transpositions that are not known in the current database.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30137425), [DOI](http://doi.org/10.1093/NAR/GKY685)

---
> ### 20- Deep genome annotation of the opportunistic human pathogen Streptococcus pneumoniae D39.
> ###### Nucleic acids research, Date: 11/2018

> #### Slager Jelle
> ##### A precise understanding of the genomic organization into transcriptional units and their regulation is essential for our comprehension of opportunistic human pathogens and how they cause disease. Using single-molecule real-time (PacBio) sequencing we unambiguously determined the genome sequence of Streptococcus pneumoniae strain D39 and revealed several inversions previously undetected by short-read sequencing. Significantly, a chromosomal inversion results in antigenic variation of PhtD, an important surface-exposed virulence factor. We generated a new genome annotation using automated tools, followed by manual curation, reflecting the current knowledge in the field. By combining sequence-driven terminator prediction, deep paired-end transcriptome sequencing and enrichment of primary transcripts by Cappable-Seq, we mapped 1015 transcriptional start sites and 748 termination sites. We show that the pneumococcal transcriptional landscape is complex and includes many secondary, antisense and internal promoters. Using this new genomic map, we identified several new small RNAs (sRNAs), RNA switches (including sixteen previously misidentified as sRNAs), and antisense RNAs. In total, we annotated 89 new protein-encoding genes, 34 sRNAs and 165 pseudogenes, bringing the S. pneumoniae D39 repertoire to 2146 genetic elements. We report operon structures and observed that 9% of operons are leaderless. The genome data are accessible in an online resource called PneumoBrowse (https://veeninglab.com/pneumobrowse) providing one of the most complete inventories of a bacterial genome to date. PneumoBrowse will accelerate pneumococcal research and the development of new prevention and treatment strategies.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30107613), [DOI](http://doi.org/10.1093/NAR/GKY725)

---
> ### 21- CRISPR-Cas adaptation in Escherichia coli requires RecBCD helicase but not nuclease activity, is independent of homologous recombination, and is antagonized by 5' ssDNA exonucleases.
> ###### Nucleic acids research, Date: 11/2018

> #### Radovcic Marin
> ##### Prokaryotic adaptive immunity is established against mobile genetic elements (MGEs) by 'naïve adaptation' when DNA fragments from a newly encountered MGE are integrated into CRISPR-Cas systems. In Escherichia coli, DNA integration catalyzed by Cas1-Cas2 integrase is well understood in mechanistic and structural detail but much less is known about events prior to integration that generate DNA for capture by Cas1-Cas2. Naïve adaptation in E. coli is thought to depend on the DNA helicase-nuclease RecBCD for generating DNA fragments for capture by Cas1-Cas2. The genetics presented here show that naïve adaptation does not require RecBCD nuclease activity but that helicase activity may be important. RecA loading by RecBCD inhibits adaptation explaining previously observed adaptation phenotypes that implicated RecBCD nuclease activity. Genetic analysis of other E. coli nucleases and naïve adaptation revealed that 5' ssDNA tailed DNA molecules promote new spacer acquisition. We show that purified E. coli Cas1-Cas2 complex binds to and nicks 5' ssDNA tailed duplexes and propose that E. coli Cas1-Cas2 nuclease activity on such DNA structures supports naïve adaptation.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30189098), [DOI](http://doi.org/10.1093/NAR/GKY799)

---
> ### 22- Rapid high-resolution measurement of DNA replication timing by droplet digital PCR.
> ###### Nucleic acids research, Date: 11/2018

> #### Batrakou Dzmitry G
> ##### Genomes are replicated in a reproducible temporal pattern. Current methods for assaying allele replication timing are time consuming and/or expensive. These include high-throughput sequencing which can be used to measure DNA copy number as a proxy for allele replication timing. Here, we use droplet digital PCR to study DNA replication timing at multiple loci in budding yeast and human cells. We establish that the method has temporal and spatial resolutions comparable to the high-throughput sequencing approaches, while being faster than alternative locus-specific methods. Furthermore, the approach is capable of allele discrimination. We apply this method to determine relative replication timing across timing transition zones in cultured human cells. Finally, multiple samples can be analysed in parallel, allowing us to rapidly screen kinetochore mutants for perturbation to centromere replication timing. Therefore, this approach is well suited to the study of locus-specific replication and the screening of cis- and trans-acting mutants to identify mechanisms that regulate local genome replication timing.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29986073), [DOI](http://doi.org/10.1093/NAR/GKY590)

---
> ### 23- clusterExperiment and RSEC: A Bioconductor package and framework for clustering of single-cell and other large gene expression datasets.
> ###### PLoS computational biology, Date: 11/2018

> #### Risso Davide
> ##### Clustering of genes and/or samples is a common task in gene expression analysis. The goals in clustering can vary, but an important scenario is that of finding biologically meaningful subtypes within the samples. This is an application that is particularly appropriate when there are large numbers of samples, as in many human disease studies. With the increasing popularity of single-cell transcriptome sequencing (RNA-Seq), many more controlled experiments on model organisms are similarly creating large gene expression datasets with the goal of detecting previously unknown heterogeneity within cells. It is common in the detection of novel subtypes to run many clustering algorithms, as well as rely on subsampling and ensemble methods to improve robustness. We introduce a Bioconductor R package, clusterExperiment, that implements a general and flexible strategy we entitle Resampling-based Sequential Ensemble Clustering (RSEC). RSEC enables the user to easily create multiple, competing clusterings of the data based on different techniques and associated tuning parameters, including easy integration of resampling and sequential clustering, and then provides methods for consolidating the multiple clusterings into a final consensus clustering. The package is modular and allows the user to separately apply the individual components of the RSEC procedure, i.e., apply multiple clustering algorithms, create a consensus clustering or choose tuning parameters, and merge clusters. Additionally, clusterExperiment provides a variety of visualization tools for the clustering process, as well as methods for the identification of possible cluster signatures or biomarkers. The R package clusterExperiment is publicly available through the Bioconductor Project, with a detailed manual (vignette) as well as well documented help pages for each function.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30180157), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006378)

---
> ### 24- SIG-DB: Leveraging homomorphic encryption to securely interrogate privately held genomic databases.
> ###### PLoS computational biology, Date: 11/2018

> #### Titus Alexander J
> ##### Genomic data are becoming increasingly valuable as we develop methods to utilize the information at scale and gain a greater understanding of how genetic information relates to biological function. Advances in synthetic biology and the decreased cost of sequencing are increasing the amount of privately held genomic data. As the quantity and value of private genomic data grows, so does the incentive to acquire and protect such data, which creates a need to store and process these data securely. We present an algorithm for the Secure Interrogation of Genomic DataBases (SIG-DB). The SIG-DB algorithm enables databases of genomic sequences to be searched with an encrypted query sequence without revealing the query sequence to the Database Owner or any of the database sequences to the Querier. SIG-DB is the first application of its kind to take advantage of locality-sensitive hashing and homomorphic encryption to allow generalized sequence-to-sequence comparisons of genomic data.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30180163), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006454)

---
> ### 25- Comparing two classes of biological distribution systems using network analysis.
> ###### PLoS computational biology, Date: 11/2018

> #### Papadopoulos Lia
> ##### Distribution networks-from vasculature to urban transportation pathways-are spatially embedded networks that must route resources efficiently in the face of pressures induced by the costs of building and maintaining network infrastructure. Such requirements are thought to constrain the topological and spatial organization of these systems, but at the same time, different kinds of distribution networks may exhibit variable architectural features within those general constraints. In this study, we use methods from network science to compare and contrast two classes of biological transport networks: mycelial fungi and vasculature from the surface of rodent brains. These systems differ in terms of their growth and transport mechanisms, as well as the environments in which they typically exist. Though both types of networks have been studied independently, the goal of this study is to quantify similarities and differences in their network designs. We begin by characterizing the structural backbone of these systems with a collection of measures that assess various kinds of network organization across topological and spatial scales, ranging from measures of loop density, to those that quantify connected pathways between different network regions, and hierarchical organization. Most importantly, we next carry out a network analysis that directly considers the spatial embedding and properties especially relevant to the function of distribution systems. We find that although both the vasculature and mycelia are highly constrained planar networks, there are clear distinctions in how they balance tradeoffs in network measures of wiring length, efficiency, and robustness. While the vasculature appears well organized for low cost, but relatively high efficiency, the mycelia tend to form more expensive but in turn more robust networks. As a whole, this work demonstrates the utility of network-based methods to identify both common features and variations in the network structure of different classes of biological transport systems.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30192745), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006428)

---
> ### 26- A Markov chain for numerical chromosomal instability in clonally expanding populations.
> ###### PLoS computational biology, Date: 11/2018

> #### Elizalde Sergi
> ##### Cancer cells frequently undergo chromosome missegregation events during mitosis, whereby the copies of a given chromosome are not distributed evenly among the two daughter cells, thus creating cells with heterogeneous karyotypes. A stochastic model tracing cellular karyotypes derived from clonal populations over hundreds of generations was recently developed and experimentally validated, and it was capable of predicting favorable karyotypes frequently observed in cancer. Here, we construct and study a Markov chain that precisely describes karyotypic evolution during clonally expanding cancer cell populations. The Markov chain allows us to directly predict the distribution of karyotypes and the expected size of the tumor after many cell divisions without resorting to computationally expensive simulations. We determine the limiting karyotype distribution of an evolving tumor population, and quantify its dependency on several key parameters including the initial karyotype of the founder cell, the rate of whole chromosome missegregation, and chromosome-specific cell viability. Using this model, we confirm the existence of an optimal rate of chromosome missegregation probabilities that maximizes karyotypic heterogeneity, while minimizing the occurrence of nullisomy. Interestingly, karyotypic heterogeneity is significantly more dependent on chromosome missegregation probabilities rather than the number of cell divisions, so that maximal heterogeneity can be reached rapidly (within a few hundred generations of cell division) at chromosome missegregation rates commonly observed in cancer cell lines. Conversely, at low missegregation rates, heterogeneity is constrained even after thousands of cell division events. This leads us to conclude that chromosome copy number heterogeneity is primarily constrained by chromosome missegregation rates and the risk for nullisomy and less so by the age of the tumor. This model enables direct integration of karyotype information into existing models of tumor evolution based on somatic mutations.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30204765), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006447)

---
> ### 27- An information theoretic treatment of sequence-to-expression modeling.
> ###### PLoS computational biology, Date: 11/2018

> #### Khajouei Farzaneh
> ##### Studying a gene's regulatory mechanisms is a tedious process that involves identification of candidate regulators by transcription factor (TF) knockout or over-expression experiments, delineation of enhancers by reporter assays, and demonstration of direct TF influence by site mutagenesis, among other approaches. Such experiments are often chosen based on the biologist's intuition, from several testable hypotheses. We pursue the goal of making this process systematic by using ideas from information theory to reason about experiments in gene regulation, in the hope of ultimately enabling rigorous experiment design strategies. For this, we make use of a state-of-the-art mathematical model of gene expression, which provides a way to formalize our current knowledge of cis- as well as trans- regulatory mechanisms of a gene. Ambiguities in such knowledge can be expressed as uncertainties in the model, which we capture formally by building an ensemble of plausible models that fit the existing data and defining a probability distribution over the ensemble. We then characterize the impact of a new experiment on our understanding of the gene's regulation based on how the ensemble of plausible models and its probability distribution changes when challenged with results from that experiment. This allows us to assess the 'value' of the experiment retroactively as the reduction in entropy of the distribution (information gain) resulting from the experiment's results. We fully formalize this novel approach to reasoning about gene regulation experiments and use it to evaluate a variety of perturbation experiments on two developmental genes of D. melanogaster. We also provide objective and 'biologist-friendly' descriptions of the information gained from each such experiment. The rigorously defined information theoretic approaches presented here can be used in the future to formulate systematic strategies for experiment design pertaining to studies of gene regulatory mechanisms.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30256780), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006459)

---
> ### 28- Wrangling distributed computing for high-throughput environmental science: An introduction to HTCondor.
> ###### PLoS computational biology, Date: 11/2018

> #### Erickson Richard A
> ##### Biologists and environmental scientists now routinely solve computational problems that were unimaginable a generation ago. Examples include processing geospatial data, analyzing -omics data, and running large-scale simulations. Conventional desktop computing cannot handle these tasks when they are large, and high-performance computing is not always available nor the most appropriate solution for all computationally intense problems. High-throughput computing (HTC) is one method for handling computationally intense research. In contrast to high-performance computing, which uses a single "supercomputer," HTC can distribute tasks over many computers (e.g., idle desktop computers, dedicated servers, or cloud-based resources). HTC facilities exist at many academic and government institutes and are relatively easy to create from commodity hardware. Additionally, consortia such as Open Science Grid facilitate HTC, and commercial entities sell cloud-based solutions for researchers who lack HTC at their institution. We provide an introduction to HTC for biologists and environmental scientists. Our examples from biology and the environmental sciences use HTCondor, an open source HTC system.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30281592), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006468)

---
> ### 29- meaRtools: An R package for the analysis of neuronal networks recorded on microelectrode arrays.
> ###### PLoS computational biology, Date: 11/2018

> #### Gelfman Sahar
> ##### Here we present an open-source R package 'meaRtools' that provides a platform for analyzing neuronal networks recorded on Microelectrode Arrays (MEAs). Cultured neuronal networks monitored with MEAs are now being widely used to characterize in vitro models of neurological disorders and to evaluate pharmaceutical compounds. meaRtools provides core algorithms for MEA spike train analysis, feature extraction, statistical analysis and plotting of multiple MEA recordings with multiple genotypes and treatments. meaRtools functionality covers novel solutions for spike train analysis, including algorithms to assess electrode cross-correlation using the spike train tiling coefficient (STTC), mutual information, synchronized bursts and entropy within cultured wells. Also integrated is a solution to account for bursts variability originating from mixed-cell neuronal cultures. The package provides a statistical platform built specifically for MEA data that can combine multiple MEA recordings and compare extracted features between different genetic models or treatments. We demonstrate the utilization of meaRtools to successfully identify epilepsy-like phenotypes in neuronal networks from Celf4 knockout mice. The package is freely available under the GPL license (GPL> = 3) and is updated frequently on the CRAN web-server repository. The package, along with full documentation can be downloaded from: https://cran.r-project.org/web/packages/meaRtools/.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30273353), [DOI](http://doi.org/10.1371/JOURNAL.PCBI.1006506)

---
> ### 30- Secure large-scale genome data storage and query.
> ###### Computer methods and programs in biomedicine, Date: 11/2018

> #### Chen Luyao
> ##### Cloud computing plays a vital role in big data science with its scalable and cost-efficient architecture. Large-scale genome data storage and computations would benefit from using these latest cloud computing infrastructures, to save cost and speedup discoveries. However, due to the privacy and security concerns, data owners are often disinclined to put sensitive data in a public cloud environment without enforcing some protective measures. An ideal solution is to develop secure genome database that supports encrypted data deposition and query.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30337067), [DOI](http://doi.org/10.1016/J.CMPB.2018.08.007)

---
> ### 31- CUILESS2016: a clinical corpus applying compositional normalization of text mentions.
> ###### Journal of biomedical semantics, Date: 11/2018

> #### Osborne John D
> ##### Traditionally text mention normalization corpora have normalized concepts to single ontology identifiers ("pre-coordinated concepts"). Less frequently, normalization corpora have used concepts with multiple identifiers ("post-coordinated concepts") but the additional identifiers have been restricted to a defined set of relationships to the core concept. This approach limits the ability of the normalization process to express semantic meaning. We generated a freely available corpus using post-coordinated concepts without a defined set of relationships that we term "compositional concepts" to evaluate their use in clinical text.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29316970), [DOI](http://doi.org/10.1186/S13326-017-0173-6)

---
> ### 32- Deep learning meets ontologies: experiments to anchor the cardiovascular disease ontology in the biomedical literature.
> ###### Journal of biomedical semantics, Date: 11/2018

> #### Arguello Casteleiro Mercedes
> ##### Automatic identification of term variants or acceptable alternative free-text terms for gene and protein names from the millions of biomedical publications is a challenging task. Ontologies, such as the Cardiovascular Disease Ontology (CVDO), capture domain knowledge in a computational form and can provide context for gene/protein names as written in the literature. This study investigates: 1) if word embeddings from Deep Learning algorithms can provide a list of term variants for a given gene/protein of interest; and 2) if biological knowledge from the CVDO can improve such a list without modifying the word embeddings created.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29650041), [DOI](http://doi.org/10.1186/S13326-018-0181-1)

---
> ### 33- Disease mentions in airport and hospital geolocations expose dominance of news events for disease concerns.
> ###### Journal of biomedical semantics, Date: 11/2018

> #### Barros Joana M
> ##### In recent years, Twitter has been applied to monitor diseases through its facility to monitor users' comments and concerns in real-time. The analysis of tweets for disease mentions should reflect not only user specific concerns but also disease outbreaks. This requires the use of standard terminological resources and can be focused on selected geographic locations. In our study, we differentiate between hospital and airport locations to better distinguish disease outbreaks from background mentions of disease concerns.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29895320), [DOI](http://doi.org/10.1186/S13326-018-0186-9)

---
> ### 34- Adverse event detection by integrating twitter data and VAERS.
> ###### Journal of biomedical semantics, Date: 11/2018

> #### Wang Junxiang
> ##### Vaccine has been one of the most successful public health interventions to date. However, vaccines are pharmaceutical products that carry risks so that many adverse events (AEs) are reported after receiving vaccines. Traditional adverse event reporting systems suffer from several crucial challenges including poor timeliness. This motivates increasing social media-based detection systems, which demonstrate successful capability to capture timely and prevalent disease information. Despite these advantages, social media-based AE detection suffers from serious challenges such as labor-intensive labeling and class imbalance of the training data.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29925405), [DOI](http://doi.org/10.1186/S13326-018-0184-Y)

---
> ### 35- Biomedical ontology alignment: an approach based on representation learning.
> ###### Journal of biomedical semantics, Date: 11/2018

> #### Kolyvakis Prodromos
> ##### While representation learning techniques have shown great promise in application to a number of different NLP tasks, they have had little impact on the problem of ontology matching. Unlike past work that has focused on feature engineering, we present a novel representation learning approach that is tailored to the ontology matching task. Our approach is based on embedding ontological terms in a high-dimensional Euclidean space. This embedding is derived on the basis of a novel phrase retrofitting strategy through which semantic similarity information becomes inscribed onto fields of pre-trained word vectors. The resulting framework also incorporates a novel outlier detection mechanism based on a denoising autoencoder that is shown to improve performance.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30111369), [DOI](http://doi.org/10.1186/S13326-018-0187-8)

---
> ### 36- OpenBiodiv-O: ontology of the OpenBiodiv knowledge management system.
> ###### Journal of biomedical semantics, Date: 11/2018

> #### Senderov Viktor
> ##### The biodiversity domain, and in particular biological taxonomy, is moving in the direction of semantization of its research outputs. The present work introduces OpenBiodiv-O, the ontology that serves as the basis of the OpenBiodiv Knowledge Management System. Our intent is to provide an ontology that fills the gaps between ontologies for biodiversity resources, such as DarwinCore-based ontologies, and semantic publishing ontologies, such as the SPAR Ontologies. We bridge this gap by providing an ontology focusing on biological taxonomy.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29347997), [DOI](http://doi.org/10.1186/S13326-017-0174-5)

---
> ### 37- Easing the burden of code review.
> ###### Nature methods, Date: 09/2018

> #### No Author
> ##### No AbstractText

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30171242), [DOI](http://doi.org/10.1038/S41592-018-0137-5)

---
> ### 38- Predicting Cancer Drug Response using a Recommender System.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Suphavilai Chayaporn
> ##### As we move toward an era of precision medicine, the ability to predict patient-specific drug responses in cancer based on molecular information such as gene expression data represents both an opportunity and a challenge. In particular, methods are needed that can accommodate the high-dimensionality of data to learn interpretable models capturing drug response mechanisms, as well as providing robust predictions across datasets.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29868820), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY452)

---
> ### 39- Experimental design and data analysis of Ago-RIP-Seq experiments for the identification of microRNA targets.
> ###### Briefings in bioinformatics, Date: 11/2018

> #### Tichy Diana
> ##### The identification of microRNA (miRNA) target genes is crucial for understanding miRNA function. Many methods for the genome-wide miRNA target identification have been developed in recent years; however, they have several limitations including the dependence on low-confident prediction programs and artificial miRNA manipulations. Ago-RNA immunoprecipitation combined with high-throughput sequencing (Ago-RIP-Seq) is a promising alternative. However, appropriate statistical data analysis algorithms taking into account the experimental design and the inherent noise of such experiments are largely lacking.Here, we investigate the experimental design for Ago-RIP-Seq and examine biostatistical methods to identify de novo miRNA target genes. Statistical approaches considered are either based on a negative binomial model fit to the read count data or applied to transformed data using a normal distribution-based generalized linear model. We compare them by a real data simulation study using plasmode data sets and evaluate the suitability of the approaches to detect true miRNA targets by sensitivity and false discovery rates. Our results suggest that simple approaches like linear regression models on (appropriately) transformed read count data are preferable.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/28379479), [DOI](http://doi.org/10.1093/BIB/BBX032)

---
> ### 40- High throughput sequencing revolution reveals conserved fundamentals of U-indel editing.
> ###### Wiley interdisciplinary reviews. RNA, Date: 06/2018

> #### Zimmer Sara L
> ##### Among Euglenozoans, mitochondrial RNA editing occurs in the diplonemids and in the kinetoplastids that include parasitic trypanosomes. Yet U-indel editing, in which open reading frames (ORFs) on mRNAs are generated by insertion and deletion of uridylates in locations dictated by guide RNAs, appears confined to kinetoplastids. The nature of guide RNA and edited mRNA populations has been cursorily explored in a surprisingly extensive number of species over the years, although complete sets of fully edited mRNAs for most kinetoplast genomes are largely missing. Now, however, high throughput sequencing technologies have had an enormous impact on what we know and will learn about the mechanisms, benefits, and final edited products of U-indel editing. Tools including PARERS, TREAT, and T-Aligner function to organize and make sense of U-indel mRNA transcriptomes, which are comprised of mRNAs harboring uridylate indels both consistent and inconsistent with translatable products. From high throughput sequencing data come arguments that partially edited mRNAs containing "junction regions" of noncanonical editing are editing intermediates, and conversely, arguments that they are dead-end products. These data have also revealed that the percent of a given transcript population that is fully or partially edited varies dramatically between transcripts and organisms. Outstanding questions that are being addressed include the prevalence of sequences that apparently encode alternative ORFs, diversity of editing events in ORF termini and 5' and 3' untranslated regions, and the differences that exist in this byzantine process between species. High throughput sequencing technologies will also undoubtedly be harnessed to probe U-indel editing's evolutionary origins. This article is categorized under: RNA Processing > RNA Editing and Modification RNA Evolution and Genomics > Computational Analyses of RNA.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29888550), [DOI](http://doi.org/10.1002/WRNA.1487)

---
> ### 41- Diagnosis of urinary tract infection based on artificial intelligence methods.
> ###### Computer methods and programs in biomedicine, Date: 11/2018

> #### Ozkan Ilker Ali
> ##### Urinary tract infection (UTI) is a common disease affecting the vast majority of people. UTI involves a simple infection caused by urinary tract inflammation as well as a complicated infection that may be caused by an inflammation of other urinary tract organs. Since all of these infections have similar symptoms, it is difficult to identify the cause of primary infection. Therefore, it is not easy to diagnose a UTI with routine examination procedures. Invasive methods that require surgery could be necessary. This study aims to develop an artificial intelligence model to support the diagnosis of UTI with complex symptoms.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30415718), [DOI](http://doi.org/10.1016/J.CMPB.2018.10.007)

---
> ### 42- Fast unsupervised nuclear segmentation and classification scheme for automatic allred cancer scoring in immunohistochemical breast tissue images.
> ###### Computer methods and programs in biomedicine, Date: 10/2018

> #### Mouelhi Aymen
> ##### This paper presents an improved scheme able to perform accurate segmentation and classification of cancer nuclei in immunohistochemical (IHC) breast tissue images in order to provide quantitative evaluation of estrogen or progesterone (ER/PR) receptor status that will assist pathologists in cancer diagnostic process.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30337080), [DOI](http://doi.org/10.1016/J.CMPB.2018.08.005)

---
> ### 43- The evolution of posttranscriptional regulation.
> ###### Wiley interdisciplinary reviews. RNA, Date: 05/2018

> #### Schaefke Bernhard
> ##### "DNA makes RNA makes protein." After transcription, mRNAs undergo a series of intertwining processes to be finally translated into functional proteins. The "posttranscriptional" regulation (PTR) provides cells an extended option to fine-tune their proteomes. To meet the demands of complex organism development and the appropriate response to environmental stimuli, every step in these processes needs to be finely regulated. Moreover, changes in these regulatory processes are important driving forces underlying the evolution of phenotypic differences across different species. The major PTR mechanisms discussed in this review include the regulation of splicing, polyadenylation, decay, and translation. For alternative splicing and polyadenylation, we mainly discuss their evolutionary dynamics and the genetic changes underlying the regulatory differences in cis-elements versus trans-factors. For mRNA decay and translation, which, together with transcription, determine the cellular RNA or protein abundance, we focus our discussion on how their divergence coordinates with transcriptional changes to shape the evolution of gene expression. Then to highlight the importance of PTR in the evolution of higher complexity, we focus on their roles in two major phenomena during eukaryotic evolution: the evolution of multicellularity and the division of labor between different cell types and tissues; and the emergence of diverse, often highly specialized individual phenotypes, especially those concerning behavior in eusocial insects. This article is categorized under: RNA Evolution and Genomics > RNA and Ribonucleoprotein Evolution Translation > Translation Regulation RNA Processing > Splicing Regulation/Alternative Splicing.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29851258), [DOI](http://doi.org/10.1002/WRNA.1485)

---
> ### 44- An improved support vector machine-based diabetic readmission prediction.
> ###### Computer methods and programs in biomedicine, Date: 11/2018

> #### Cui Shaoze
> ##### In healthcare systems, the cost of unplanned readmission accounts for a large proportion of total hospital payment. Hospital-specific readmission rate becomes a critical issue around the world. Quantification and early identification of unplanned readmission risks will improve the quality of care during hospitalization and reduce the occurrence of readmission. In clinical practice, medical workers generally use LACE score method to evaluate patient readmission risks, but this method usually performs poorly. With this in mind, this study presents a novel method combining support vector machine and genetic algorithm to build the risk prediction model, which simultaneously involves feature selection and the processing of imbalanced data. This model aims to provide decision support for clinicians during the discharge management of patients with diabetes.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30415712), [DOI](http://doi.org/10.1016/J.CMPB.2018.10.012)

---
> ### 45- Cas9 versus Cas12a/Cpf1: Structure-function comparisons and implications for genome editing.
> ###### Wiley interdisciplinary reviews. RNA, Date: 05/2018

> #### Swarts Daan C
> ##### Cas9 and Cas12a are multidomain CRISPR-associated nucleases that can be programmed with a guide RNA to bind and cleave complementary DNA targets. The guide RNA sequence can be varied, making these effector enzymes versatile tools for genome editing and gene regulation applications. While Cas9 is currently the best-characterized and most widely used nuclease for such purposes, Cas12a (previously named Cpf1) has recently emerged as an alternative for Cas9. Cas9 and Cas12a have distinct evolutionary origins and exhibit different structural architectures, resulting in distinct molecular mechanisms. Here we compare the structural and mechanistic features that distinguish Cas9 and Cas12a, and describe how these features modulate their activity. We discuss implications for genome editing, and how they may influence the choice of Cas9 or Cas12a for specific applications. Finally, we review recent studies in which Cas12a has been utilized as a genome editing tool. This article is categorized under: RNA Interactions with Proteins and Other Molecules > Protein-RNA Interactions: Functional Implications Regulatory RNAs/RNAi/Riboswitches > Biogenesis of Effector Small RNAs RNA Interactions with Proteins and Other Molecules > RNA-Protein Complexes.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29790280), [DOI](http://doi.org/10.1002/WRNA.1481)

---
> ### 46- A semi-supervised deep learning method based on stacked sparse auto-encoder for cancer prediction using RNA-seq data.
> ###### Computer methods and programs in biomedicine, Date: 11/2018

> #### Xiao Yawen
> ##### Cancer has become a complex health problem due to its high mortality. Over the past few decades, with the rapid development of the high-throughput sequencing technology and the application of various machine learning methods, remarkable progress in cancer research has been made based on gene expression data. At the same time, a growing amount of high-dimensional data has been generated, such as RNA-seq data, which calls for superior machine learning methods able to deal with mass data effectively in order to make accurate treatment decision.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30415723), [DOI](http://doi.org/10.1016/J.CMPB.2018.10.004)

---
> ### 47- DeepMirTar: a deep-learning approach for predicting human miRNA targets.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Wen Ming
> ##### MicroRNAs (miRNAs) are small non-coding RNAs that function in RNA silencing and post-transcriptional regulation of gene expression by targeting messenger RNAs (mRNAs). Because the underlying mechanisms associated with miRNA binding to mRNA are not fully understood, a major challenge of miRNA studies involves the identification of miRNA-target sites on mRNA. In silico prediction of miRNA-target sites can expedite costly and time-consuming experimental work by providing the most promising miRNA-target-site candidates.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29868708), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY424)

---
> ### 48- Guide Swap enables genome-scale pooled CRISPR-Cas9 screening in human primary cells.
> ###### Nature methods, Date: 11/2018

> #### Ting Pamela Y
> ##### CRISPR-Cas9 screening allows genome-wide interrogation of gene function. Currently, to achieve the high and uniform Cas9 expression desirable for screening, one needs to engineer stable and clonal Cas9-expressing cells-an approach that is not applicable in human primary cells. Guide Swap permits genome-scale pooled CRISPR-Cas9 screening in human primary cells by exploiting the unexpected finding that editing by lentivirally delivered, targeted guide RNAs (gRNAs) occurs efficiently when Cas9 is introduced in complex with nontargeting gRNA. We validated Guide Swap in depletion and enrichment screens in CD4

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30297964), [DOI](http://doi.org/10.1038/S41592-018-0149-1)

---
> ### 49- Gaining insight into transcriptome-wide RNA population dynamics through the chemistry of 4-thiouridine.
> ###### Wiley interdisciplinary reviews. RNA, Date: 11/2018

> #### Duffy Erin E
> ##### Cellular RNA levels are the result of a juggling act between RNA transcription, processing, and degradation. By tuning one or more of these parameters, cells can rapidly alter the available pool of transcripts in response to stimuli. While RNA sequencing (RNA-seq) is a vital method to quantify RNA levels genome-wide, it is unable to capture the dynamics of different RNA populations at steady-state or distinguish between different mechanisms that induce changes to the steady-state (i.e., altered rate of transcription vs. degradation). The dynamics of different RNA populations can be studied by targeted incorporation of noncanonical nucleosides. 4-Thiouridine (s

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30370679), [DOI](http://doi.org/10.1002/WRNA.1513)

---
> ### 50- Antisense and yet sensitive: Copy number control of rolling circle-replicating plasmids by small RNAs.
> ###### Wiley interdisciplinary reviews. RNA, Date: 10/2018

> #### Pluta Radoslaw
> ##### Bacterial plasmids constitute a wealth of shared DNA amounting to about 20% of the total prokaryotic pangenome. Plasmids replicate autonomously and control their replication by maintaining a fairly constant number of copies within a given host. Plasmids should acquire a good fitness to their hosts so that they do not constitute a genetic load. Here we review some basic concepts in plasmid biology, pertaining to the control of replication and distribution of plasmid copies among daughter cells. A particular class of plasmids is constituted by those that replicate by the rolling circle mode (rolling circle-replicating [RCR]-plasmids). They are small double-stranded DNA molecules, with a rather high number of copies in the original host. RCR-plasmids control their replication by means of a small short-lived antisense RNA, alone or in combination with a plasmid-encoded transcriptional repressor protein. Two plasmid prototypes have been studied in depth, namely the staphylococcal plasmid pT181 and the streptococcal plasmid pMV158, each corresponding to the two types of replication control circuits, respectively. We further discuss possible applications of the plasmid-encoded antisense RNAs and address some future directions that, in our opinion, should be pursued in the study of these small molecules. This article is categorized under: Regulatory RNAs/RNAi/Riboswitches > Regulatory RNAs RNA Structure and Dynamics > Influence of RNA Structure in Biological Systems.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30074293), [DOI](http://doi.org/10.1002/WRNA.1500)

---
> ### 51- Single-cell genomics to guide human stem cell and tissue engineering.
> ###### Nature methods, Date: 09/2018

> #### Camp J Gray
> ##### To understand human development and disease, as well as to regenerate damaged tissues, scientists are working to engineer certain cell types in vitro and to create 3D microenvironments in which cells behave physiologically. Single-cell genomics (SCG) technologies are being applied to primary human organs and to engineered cells and tissues to generate atlases of cell diversity in these systems at unparalleled resolution. Moving beyond atlases, SCG methods are powerful tools for gaining insight into the engineering and disease process. Here we discuss how scientists can use single-cell sequencing to optimize human cell and tissue engineering by measuring precision, detecting inefficiencies, and assessing accuracy. We also provide a perspective on how emerging SCG methods can be used to reverse-engineer human cells and tissues and unravel disease mechanisms.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30171231), [DOI](http://doi.org/10.1038/S41592-018-0113-0)

---
> ### 52- A hybrid data mining model for diagnosis of patients with clinical suspicion of dementia.
> ###### Computer methods and programs in biomedicine, Date: 10/2018

> #### Moreira Leonard Barreto
> ##### Given the phenomenon of aging population, dementias arise as a complex health problem throughout the world. Several methods of machine learning have been applied to the task of predicting dementias. Given its diagnostic complexity, the great challenge lies in distinguishing patients with some type of dementia from healthy people. Particularly in the early stages, the diagnosis positively impacts the quality of life of both the patient and the family. This work presents a hybrid data mining model, involving the mining of texts integrated to the mining of structured data. This model aims to assist specialists in the diagnosis of patients with clinical suspicion of dementia.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30337069), [DOI](http://doi.org/10.1016/J.CMPB.2018.08.016)

---
> ### 53- LncRNAnet: long non-coding RNA identification using deep learning.
> ###### Bioinformatics (Oxford, England), Date: 11/2018

> #### Baek Junghwan
> ##### Long non-coding RNAs (lncRNAs) are important regulatory elements in biological processes. LncRNAs share similar sequence characteristics with messenger RNAs, but they play completely different roles, thus providing novel insights for biological studies. The development of next-generation sequencing has helped in the discovery of lncRNA transcripts. However, the experimental verification of numerous transcriptomes is time consuming and costly. To alleviate these issues, a computational approach is needed to distinguish lncRNAs from the transcriptomes.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29850775), [DOI](http://doi.org/10.1093/BIOINFORMATICS/BTY418)

---
> ### 54- Single-cell RNA-seq reveals dynamic transcriptome profiling in human early neural differentiation.
> ###### GigaScience, Date: 11/2018

> #### Shang Zhouchun
> ##### Investigating cell fate decision and subpopulation specification in the context of the neural lineage is fundamental to understanding neurogenesis and neurodegenerative diseases. The differentiation process of neural-tube-like rosettes in vitro is representative of neural tube structures, which are composed of radially organized, columnar epithelial cells and give rise to functional neural cells. However, the underlying regulatory network of cell fate commitment during early neural differentiation remains elusive.

> ##### Link to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30239706), [DOI](http://doi.org/10.1093/GIGASCIENCE/GIY117)

---
> DOIs to be checked manually:
> ##### 55- http://doi.org/10.1101/474403
> ##### 56- http://doi.org/10.1101/473488
> ##### 57- http://doi.org/10.1002/ASI.24042
> ##### 58- http://doi.org/10.1101/466151
> ##### 59- http://doi.org/10.1002/ASI.24073
> ##### 60- http://doi.org/10.1101/310599
> ##### 61- http://doi.org/10.1101/472464
> ##### 62- http://doi.org/10.1101/474536
> ##### 63- http://doi.org/10.1002/ASI.24047
> ##### 64- http://doi.org/10.1101/321539
> ##### 65- http://doi.org/10.1101/468025
> ##### 66- http://doi.org/10.1101/470195
> ##### 67- http://doi.org/10.1101/475426
> ##### 68- http://doi.org/10.1101/474767
> ##### 69- http://doi.org/10.1002/ASI.24033
> ##### 70- http://doi.org/10.1101/472423
> ##### 71- http://doi.org/10.1002/ASI.24080
> ##### 72- http://doi.org/10.1101/469189
> ##### 73- http://doi.org/10.1101/472217
> ##### 74- http://doi.org/10.1002/ASI.24048
> ##### 75- http://doi.org/10.1101/444828
> ##### 76- http://doi.org/10.1101/470278
> ##### 77- http://doi.org/10.1101/470211
> ##### 78- http://doi.org/10.1101/470203
